src/client/views/customfunctions.html

```diff
@@ -0,0 +1,16 @@
+﻿<!DOCTYPE html>
```

Comment:
Please move both this file and the JS (or rather, inline the JS into the HTML) to live under "src/server/templates".  And add a route in "src/server.ts" to serve this file (see how the "try" route is served; it should be an easy pattern to copy-paste).